### ISSUE TITLE
Exclude files base on patterns

### ISSUE THREAD

**Issue by slifer2015** [mgechev/revive#1011]
**Exclude files base on patterns**
I want to exclude file base on pattern , for example exclude all test files

**To Reproduce**
1. I updated revive `go install github.com/mgechev/revive@latest`
2. I run it with the following flags & configuration file:

```shell
EXCLUDE_PATTERNS := \
	--exclude '**/*_test.go'

.PHONY: lint
lint:
	docker run --rm \
		-v $(PWD):/app \
		-w /app \
		ghcr.io/mgechev/revive:v1.3.7 \
		-config revive.toml \
		-formatter friendly \
		./...

```

```toml
enableAllRules = true
```

**Expected behavior**
I expected that files end of _test.go is excluded but It's not happening and still getting lint for test files
